src/components/SearchBar.js

```diff
@@ -1,14 +1,19 @@
-import React, { Component } from 'react'
-import ResultsList from './containers/ResultsList'
-import search from '../js/search'
```

Comment:
Could you please wire the search library up to your changes? We want to make sure your changes don't reduce the functionality of the app. You don't need to do anything fancy with the results in this PR (they could simply continue to print to the console), but we definitely don't want to undo work that's already been done.